tgui/packages/tgui/interfaces/AntagInfoTraitor.tsx

```diff
@@ -149,6 +158,19 @@ const UplinkSection = (props, context) => {
           </>
         )}
       </Stack>
+      <br />
+      <Section textAlign="Center">
+        If you lose your uplink you can construct a Syndicate Uplink Beacon and
```

Comment:
```suggestion
        If you lose your uplink, you can craft a Syndicate Uplink Beacon and
```

Small indicator to tell them to craft, not construct with a machine frame.

This might still be too vague for new players but at that point there's only so much you can do before it gets intrusive.

Reply:
good call, ty